storage/hostpath/security/hostpath.yaml

```diff
@@ -8,27 +8,18 @@ spec:
     seLinuxOptions:
       level: s0:c13,c2
   containers:
-  - name: a
-    image: aosqe/hello-openshift
-    ports:
-      - containerPort: 8080
-        protocol: "TCP"
-    securityContext:
-      runAsUser: 1000130001
-    volumeMounts:
-      - mountPath: "/example/hostpath/a"
-        name: host-vol
-  - name: b
-    image: aosqe/hello-openshift
+  - image: aosqe/hello-openshift
+    name: a
     ports:
       - containerPort: 8081
         protocol: "TCP"
     securityContext:
-      runAsUser: 1000130002
+    runAsUser: 1000130001
```

Comment:
Did you test before you summit/push ?

Reply:
yes

Reply:
uid is 1000130001
btw, I will remove part of steps in cucumber scripts